### PR TITLE
refs #649 ignore limit in bool columns and cast to bool defaults for postgres

### PR DESCRIPTION
**Pull request by steinkel**
Make column definition less error prone when switching from mysql or baked code.

* Ignore limit in boolean columns
* Cast to boolean value the defaults if column type is boolean

### CODE REVIEW

File: src/Phinx/Db/Adapter/PostgresAdapter.php [hunk 1]
```diff
@@ -875,12 +875,14 @@ public function dropDatabase($name)
      * @param  mixed $default
      * @return string
```
Comment (ntzm) [inline src/Phinx/Db/Adapter/PostgresAdapter.php]: Needs adding to the docblock

Comment (dereuromark) [reply src/Phinx/Db/Adapter/PostgresAdapter.php]: For return statements? I thought only params do.

Comment (ntzm) [reply src/Phinx/Db/Adapter/PostgresAdapter.php]: A new param was added

Comment (steinkel) [reply src/Phinx/Db/Adapter/PostgresAdapter.php]: fixed docblock

**Comment by lorenzo** [comment]
Thanks!

**Pull request merged**